File: programs/containers/vector.cpp
#include <stdio.h>
#include <unistd.h>
#include <time.h>
#include <stdlib.h>
#include <iostream>
#include <list>
#include <vector>
#include <algorithm>

int main()
{
    srand(time(NULL));
    std::vector<int> test;
    for (int i = 0; i < 100000000; ++i)
    {
        test.push_back(rand());
    }
    printf("Done\n");
    sleep(20);
    printf("Sorting start\n");
    std::sort(test.begin(), test.end());
    printf("Sorting Done\n");
    sleep(10);
}

File: programs/containers/vector.py
import random
import array as arr
import time
import numpy as np

def main():
    random.seed()
    test = arr.array('i', (random.randint(0, 2**31 - 1) for _ in range(100000000)))
    print(test[:10])
    print("Done")
    time.sleep(2)
    print("Sorting start")
    result = np.sort(test)
    print(result[:10])
    print("Sorting Done")
    time.sleep(1)

if __name__ == "__main__":
    main()